eva-accession-import-dbsnp2/pom.xml

```diff
@@ -59,6 +52,16 @@
             <groupId>uk.ac.ebi.eva</groupId>
             <artifactId>variation-commons-batch</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
```

Comment:
Could you let me know where the JAXB API is being used in the code?

Reply:
See: [Java lang NoClassDefFoundError JAVAX XML bind exception](https://stackoverflow.com/questions/43574426/how-to-resolve-java-lang-noclassdeffounderror-javax-xml-bind-jaxbexception-in-j)

Reply:
Stack trace for all tests have this error if that dependency goes away: 
```
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaAutoConfiguration.class]: Invocation of init method failed; nested exception is java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1628)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:555)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
  
```

Reply:
Please use JDK8, that is the version we use in production, and the one the code definitely needs to be tested against.